src/Compilers/Core/Portable/Syntax/InternalSyntax/SeparatedSyntaxListBuilder.cs

```diff
@@ -59,13 +57,18 @@ public void Clear()
             _builder.Clear();
         }
 
+        public void RemoveLast()
+        {
+            this._builder.RemoveLast();
+        }
```

Comment:
This is used here: 
https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/Portable/Syntax/InternalSyntax/SyntaxRewriter.vb#L88